tests/viz_tests.py

```diff
@@ -634,3 +634,109 @@ def test_rose_vis_get_data(self):
             ],
         }
         self.assertEqual(expected, res)
+
+
+class BaseDeckGLVizTestCase(unittest.TestCase):
+
+    def test_get_metrics(self):
+        form_data = {'size': 'large'}
```

Comment:
Typically a `form_data` dict will be much larger and have as many keys a specified in that particular viz type. We should talk about a way to test `viz.py` with examples that represent reality, may by sharing code with the examples, along with assertions to complete/test the examples.